server/data/courseClient.ts

```diff
@@ -14,4 +14,12 @@ export default class CourseClient {
   async all(): Promise<Array<Course>> {
     return (await this.restClient.get({ path: paths.courses.index({}) })) as Array<Course>
   }
+
+  async find(id: Course['id']): Promise<Course> {
+    return (await this.restClient.get({ path: paths.courses.show({ id }) })) as Course
+  }
+
+  async offerings(courseId: Course['id']): Promise<Array<CourseOffering>> {
```

Comment:
I think we should call this `findCourseOfferings` or something similar to keep it consistent with our "verby" function name convention in our clients.

Reply:
Yeah, I don't think it's a huge deal but I think it'd be good to aim for the verby style in both

Reply:
So we have the following client/service mappings:
- `getAll` -> `getCourses`
- `find` -> `getCourse`
- `getOfferings` -> `getOfferingsByCourse`

Is it worth bringing them fully in sync, following the service method naming convention? I guess `all` and `find` are Active Record like conventions, but if we want the clients to be verby like the services, should we go all the way with the consistency?

Reply:
I like how you've mapped them out there in your comment - can we do that?

Reply:
Yep, that's how I have it at the moment (locally), just wondering how we conceptualise the distinction

Reply:
Hmm, but if it was more Rails-y, offerings would probably be an association, in which case it would just be `course.offerings`, but then this is a client not a model 🤔

`find` is typically for a single record in Active Record, but `where` or `select` don't feel appropriate either

Maybe `findOfferings` is fine for now